distributed/deploy/ssh.py

```diff
@@ -278,13 +278,26 @@ def SSHCluster(
 
     Examples
     --------
+    The most relevant example is with a remote cluser you have SSH
+    access to as user ``foo``. Best practice is to generate a key-pair
+    following the `SSH keygen tutorial`_:
+
+    .. code:: bash
+
+       $ # Generate a key pair
+       $ ssh-keygen -t rsa -b 4096 -f ~/.ssh/dask-ssh -P ""
+       $ # Copy to remote machine
+       $ ssh-copy-id -i ~/.ssh/dask-ssh user@machine
+
+    Now it's possible to login to ``machine`` without entering a
+    password via ``ssh -i ~/.ssh-dask-ssh user@machine``.  Let's
+    create an ``SSHCluster``:
+
     >>> from dask.distributed import Client, SSHCluster
     >>> cluster = SSHCluster(
-    ...     ["localhost", "localhost", "localhost", "localhost"],
-    ...     connect_options={"known_hosts": None},
-    ...     worker_options={"nthreads": 2},
-    ...     scheduler_options={"port": 0, "dashboard_address": ":8797"}
-    ... )
+    ...     ["machine1", "machine1"],
+    ...     scheduler_options={"port": 0, "dashboard_address": ":8797"},
+    ...     connect_options={...})
```

Comment:
I would expect the `connect_options` should explicitly set the username and key location given your example above.

```suggestion
    ...     connect_options={"username": "user", "client_keys": "~/.ssh/dask-ssh"})
```